client/osgi/pom.xml

```diff
@@ -58,10 +59,7 @@
               org.slf4j.spi;version="1.2.3",
             </Import-Package>
             <Export-Package>
-              org.sonarsource.sonarlint.core.*;version="${project.version}",
-              org.sonarsource.sonarlint.shaded.com.google.gson.*;version="${gson.version}",
-              com.google.protobuf;version="${protobuf.version}",
-              org.eclipse.lsp4j.jsonrpc.*;version="${lsp4j.version}",
+              org.sonarsource.sonarlint.core.client.*;version="${project.version}",
```

Comment:
Don't we also have to export `org.sonarsource.sonarlint.core.commons.*` and `org.sonarsource.sonarlint.core.rpc.*`?
I tried using the bundle in SLE and it didn't work (mostly due to other reasons but also because when accessing stuff in both this packages Tycho complained about it not being API).